plugins/database/src/db_plugin.cpp

```diff
@@ -233,6 +233,15 @@ namespace
 
         return SUCCESS();
     } // get_auth_config
+    auto log_db_helper(const irods::error& _err) -> void
```

Comment:
I think this function can be dropped now.

Reply:
Done.